### DISCUSSION TITLE
Error P1 STM

### DISCUSSION THREAD

**Discussion by mariapenon** [TheAlbertDev/MASB#46] [Q&A] [answered]
Hi, I'm getting this error in the beginning of the STM practice 1: 
![image](https://user-images.githubusercontent.com/63785627/108692570-0f77f600-74fd-11eb-98d7-46a43a1fce39.png)
If I look at it closely, the error seems to be in the name of the folder in which I save all the practices. 
![image](https://user-images.githubusercontent.com/63785627/108692672-2d455b00-74fd-11eb-9a01-cab25aa15417.png)
Is there any way to fix it? If I change the name of the folders, will it alter my repository?
Thanks.

**Comment by Albert-Alvarez** [comment, marked as answer]
The only way to fix it is to place your repository in a path with English alphanumeric characters only. It's easy to solve. Just simply copy/paste your entire folder (`masb-p01-ecologiques`) and place it in another location free of "strange" characters into its path. For example, you can create a folder called `workspace` into your root folder. E.g., if you work on a Windows computer, create a folder on `c:\workspace\` and paste there your folder `masb-p01-ecologiques`. Then, you can work normally from your new location.

Let us know if you have been able to solve your errors 😉

**Comment by mariapenon** [comment]
The error has disappeared creating a new folder with english alphanumeric characters, and doing a copy/paste. Thank you!
